# Author: Karl Linden (lilrc) <lilrc@users.sourceforge.net>
# Purpose: x11-libs/fox does not install a fox.pc. This file is named
#          fox17.pc instead (fox slot 1.7).
# This patch has not been sent upstream.

diff --git a/configure.ac b/configure.ac
index 3533bef..7f97550 100644
--- a/configure.ac
+++ b/configure.ac
@@ -185,7 +185,7 @@ if test "x$testgui_enabled" != "xno"; then
 		fi
 	else
 		# On Linux and FreeBSD platforms, use pkg-config to find fox.
-		PKG_CHECK_MODULES([fox], [fox])
+		PKG_CHECK_MODULES([fox], [fox17])
 		LIBS_TESTGUI="${LIBS_TESTGUI} $fox_LIBS"
 		if test "x$os" = xfreebsd; then
 			LIBS_TESTGUI="${LIBS_TESTGUI} -L/usr/local/lib"
